pkg/apis/serving/v1alpha1/kfservice_framework_pytorch.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kubeflow/kfserving/pkg/utils"
+	v1 "k8s.io/api/core/v1"
+)
+
+// TODO add image name to to configmap
+var (
+	AllowedPyTorchRuntimeVersions = []string{
+		"latest",
+	}
+	InvalidPyTorchRuntimeVersionError = "RuntimeVersion must be one of " + strings.Join(AllowedPyTorchRuntimeVersions, ", ")
+	PyTorchServerImageName            = "animeshsingh/pytorchserver"
```

Comment:
I set up an autobuild for this. Can you cut me and issue and follow the convention of the other frameworks.
While testing this locally, you can go via the configmap :)

Reply:
Done

Reply:
the docker image and and gcs location need to be updated